.github/workflows/publish.yml

```diff
@@ -0,0 +1,14 @@
+jobs:
+  pypi-publish:
+    name: Upload release to PyPI
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/pywebsocket3
+    permissions:
+      id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
+    steps:
+    # retrieve your distributions here
```

Comment:
This comment is confusing to me, but maybe we need a download step like here: https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#publishing-the-distribution-to-pypi ?

Reply:
Seemed like that is the case. I have added a new set of steps to build distribution package.